base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -263,7 +263,7 @@ BEGIN
     IDS_VERTICAL "Vertical"
     IDS_PRINTRES "%d x %d pixeli/cm"
     IDS_CANTPASTE "Nu a putut fi lipit din clipboard. Formatul de date este fie incorect, fie nesuportat."
-    IDS_SAVEERROR "Nu a putut fi salvată imaginea bitmap („hartă de biți”) în fișierul:\n\n%s"
-    IDS_CANTSENDMAIL "Nu a putut fi trimisă o scrisoare."
-    IDS_LOSECOLOR "Informațiile de culoare vor fi pierdute în această operațiune. Sunteți sigur că veți continua?"
+    IDS_SAVEERROR "Eșec în salvarea imaginii bitmap („hartă de biți”) în fișierul:\n\n%s"
+    IDS_CANTSENDMAIL "Eșect în trimiterea unei scrisori."
+    IDS_LOSECOLOR "În această operațiune va fi pierdută informația legată de culoare. Sunteți sigur că veți continua?"
```

Comment:
Please read again [my previous comment](https://github.com/reactos/reactos/pull/5564#discussion_r1292894350), a bit more careful this time.

Reply:
I tried again, but I still don't know what to do with "În această operațiune va fi pierdută informația legată de culoare". I don't know how to do it gender neutral.

Reply:
The gender neutral bit was not about the sentence you quoted, it was about the next one, which you just addressed in [commit 10606af2a639a578038ad4b5e18d0ac583eb24dd](https://github.com/reactos/reactos/pull/5564/commits/10606af2a639a578038ad4b5e18d0ac583eb24dd).